### PR TITLE
Feat/User_CRUD_Operations

### PR DESCRIPTION
**Pull request by VinBid**
* Updated with CRUD Operations for Users
* Tested Operations
**BLOCKERS:**
* The Update Operation does not seem to work if I do not include all the input fields. Otherwise it works. Would be great if someone could look through.

### CODE REVIEW

File: src/typeDefs/index.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import User from "./User.js";
-import Playlist from "./Playlist.js";
-import Song from "./Song.js";
 
-export default [User, Playlist, Song];
+
```
Comment (Austin2Shih) [inline src/typeDefs/index.ts]: Make sure to get your eslint stuff properly configured. Make sure to not have linting errors or else you won't pass the checks

**Review by Austin2Shih — CHANGES_REQUESTED** [review]
If you need help, refer to Kiran's code which is in another PR at the moment. Sorry I am super busy so you might need to consult other people working on similar tasks for this. However, if ya'll run into huge roadblocks then still feel free to ask me for help.

**Review by VinBid — COMMENTED** [review]
* Fixed the Issue with the updating of users
* Cleaned up Code to prevent problems

File: prisma/dev.db
Comment (brandonw504) [inline prisma/dev.db:1]: I looked through this file and dev.db-journal, and I'm not sure what they are, did you add them intentionally? I saw stuff about SQLite but we should be using PostgreSQL, so I just wanted to check with you to make sure.

**Review by brandonw504 — COMMENTED** [review]
I tested all the CRUD operations and everything works great! I'd suggest renaming your migrations to be more descriptive than "migration" and "f" so that in the future we'll know by the title what the migrations did. Other than that, looks good to me!

File: prisma/dev.db
Comment (VinBid) [reply prisma/dev.db:1]: That was by accident, probably when I was initially testing things.

**Comment by brandonw504** [comment]
I think the last two things to do are deleting dev.db and dev.db-journal and re-running the migrations (you can just delete your migration files and re-run the migration) so that you can give them more descriptive names. After that, we should be good to merge!

**Pull request closed without merging**